use pkg-config to define libs/includes instead of old paths
remove debug compiler flag
--- a/config.mk
+++ b/config.mk
@@ -7,12 +7,11 @@ VERSION = 1
 PREFIX = /usr/local
 MANPREFIX = ${PREFIX}/share/man
 
-X11INC = /usr/X11R6/include
-X11LIB = /usr/X11R6/lib
+PKG_CONFIG ?= pkg-config
 
 # includes and libs
-INCS = -I. -I/usr/include -I/usr/include/freetype2 -I${X11INC}
-LIBS = -L/usr/lib -lc -lm -L${X11LIB} -lXft -lfontconfig -lX11
+INCS = $(shell ${PKG_CONFIG} --print-errors --exists fontconfig xft x11 --cflags)
+LIBS = -lc -lm $(shell ${PKG_CONFIG} --print-errors --exists fontconfig xft x11 --libs)
 # OpenBSD (uncomment)
 #INCS = -I. -I${X11INC} -I${X11INC}/freetype2
 # FreeBSD (uncomment)
@@ -21,8 +20,8 @@ LIBS = -L/usr/lib -lc -lm -L${X11LIB} -lXft -lfontconfig -lX11
 
 # flags
 CPPFLAGS = -DVERSION=\"${VERSION}\" -D_XOPEN_SOURCE=600
-CFLAGS += -g -std=c99 -pedantic -Wall ${INCS} ${CPPFLAGS}
-LDFLAGS += -g ${LIBS}
+CFLAGS += -std=c99 -pedantic -Wall ${INCS} ${CPPFLAGS}
+LDFLAGS += ${LIBS}
 #CFLAGS += -std=c99 -pedantic -Wall -Os ${INCS} ${CPPFLAGS}
 #LDFLAGS += ${LIBS}
 
